src/strangerThings.js

```diff
@@ -11,7 +11,10 @@ function sleep(ms) {
 TODO:
 This function should return a list of all the distinct characters in UPPERCASE that have been typed in the textbox with the id "message"
 */
-function getCharacters() {}
+function getCharacters()
+{
+	return document.querySelector("#message").value.split("");
+}
```

Comment:
The characters must be converted to uppercase too.

Reply:
Ohh okay
I would just add .map(k=>k.toUpperCase());

Reply:
Great.